joint_trajectory_controller/test/test_trajectory_actions.cpp

```diff
@@ -373,145 +396,84 @@ TEST_F(TestTrajectoryActions, test_goal_tolerances_success) {
+  EXPECT_EQ(true, common_goal_accepted_);
+  EXPECT_EQ(rclcpp_action::ResultCode::SUCCEEDED, common_resultcode_);
   EXPECT_EQ(
     control_msgs::action::FollowJointTrajectory_Result::SUCCESSFUL,
-    common_action_result_code);
+    common_action_result_code_);
 
-  EXPECT_NEAR(7.0, test_robot->pos1, common_threshold);
-  EXPECT_NEAR(8.0, test_robot->pos2, common_threshold);
-  EXPECT_NEAR(9.0, test_robot->pos3, common_threshold);
-
-  executor.cancel();
+  EXPECT_NEAR(7.0, test_robot_->pos1, COMMON_THRESHOLD);
+  EXPECT_NEAR(8.0, test_robot_->pos2, COMMON_THRESHOLD);
+  EXPECT_NEAR(9.0, test_robot_->pos3, COMMON_THRESHOLD);
 }
 
 TEST_F(TestTrajectoryActions, test_state_tolerances_fail) {
   // set joint tolerance parameters
   double state_tol = 0.0001;
-  traj_lifecycle_node->declare_parameter("constraints.joint1.trajectory", 0.0);
-  traj_lifecycle_node->declare_parameter("constraints.joint2.trajectory", 0.0);
-  traj_lifecycle_node->declare_parameter("constraints.joint3.trajectory", 0.0);
-  traj_lifecycle_node->set_parameter(rclcpp::Parameter("constraints.joint1.trajectory", state_tol));
-  traj_lifecycle_node->set_parameter(rclcpp::Parameter("constraints.joint2.trajectory", state_tol));
-  traj_lifecycle_node->set_parameter(rclcpp::Parameter("constraints.joint3.trajectory", state_tol));
-
-  rclcpp::executors::MultiThreadedExecutor executor;
-  executor.add_node(traj_lifecycle_node->get_node_base_interface());
-
-  traj_controller->on_configure(traj_lifecycle_node->get_current_state());
-  traj_controller->on_activate(traj_lifecycle_node->get_current_state());
-  SetUpActionClient();
-  executor.add_node(node->get_node_base_interface());
-
-  auto future_handle = std::async(
-    std::launch::async, [&executor]() -> void {
-      executor.spin();
-    });
-
-  auto thread_func = [&]() {
-      // controller hardware cycle update loop
-      auto start_time = rclcpp::Clock().now();
-      rclcpp::Duration wait = rclcpp::Duration::from_seconds(2.0);
-      auto end_time = start_time + wait;
-      while (rclcpp::Clock().now() < end_time) {
-        test_robot->read();
-        traj_controller->update();
-        test_robot->write();
-      }
-    };
-  std::thread controller_hw_thread(thread_func);
-
-  // common_goal_response and common_result_response
-  // sometimes doesnt receive calls when we dont sleep
-  std::this_thread::sleep_for(std::chrono::milliseconds(200));
-
-  GoalOptions opt;
-  opt.goal_response_callback =
-    std::bind(&TestTrajectoryActions::common_goal_response, this, _1);
-  opt.result_callback =
-    std::bind(&TestTrajectoryActions::common_result_response, this, _1);
-  opt.feedback_callback = nullptr;
+  traj_lifecycle_node_->declare_parameter("constraints.joint1.trajectory", 0.0);
+  traj_lifecycle_node_->declare_parameter("constraints.joint2.trajectory", 0.0);
+  traj_lifecycle_node_->declare_parameter("constraints.joint3.trajectory", 0.0);
+  traj_lifecycle_node_->set_parameter(
```

Comment:
dear lord this 4-line break-up is ugly

let's add the value setting to the `declare_parameter` call, then please try to see how we can avoid uncrustify ruining the format...

Reply:
Done.